src/server/common/dbconn.hpp

```diff
@@ -27,6 +27,7 @@
 #include <vector>
 #include <unordered_map>
 
+// make sure to update dbconn::query() as well when changing these listed values
```

Comment:
With the static assert this isn't necessary anymore. But it also doesn't hurt so keep it if you like.